hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/BasicRootedOzoneClientAdapterImpl.java

```diff
@@ -648,4 +663,20 @@ private FileStatusAdapter toFileStatusAdapter(OzoneFileStatus status) {
         status.getPath()
     );
   }
+
+  private FileStatusAdapter rootFileStatusAdapter() {
+    return new FileStatusAdapter(
```

Comment:
should we make this static final?

Reply:
Good point. We should.

Made it static. As it's private we won't need final.